README.md

```diff
@@ -1,3 +1,58 @@
 # Enigma-PD-WML
 
-Segment White Mater Lesions (WML) in T1-weighted and FLAIR MR images using FSL and U-Net
+Segment White Mater Lesions (WML) in T1-weighted and FLAIR MRI images using FSL and U-Net
+
+## Build and run the docker container
+
+- Clone this repository
+
+```bash
+git clone https://github.com/UCL-ARC/Enigma-PD-WML.git
+```
+
+- Build the docker image
+
+```bash
+cd Enigma-PD-WML
+docker build -f Dockerfile -t fsl_test .
+```
+
+- Create `code` and `data` directories inside the `Enigma-PD-WML` directory
```

Comment:
we could create an empty `code` directory with a `.gitkeep` file in it and commit it to this repo just so users don't need to create it themselves

Reply:
Good point! Although I'm wondering if we should move away from recommending creating the code/data dirs directly inside the github repo dir anyway? As we're setting each as explicit volumes here: `docker run -v "$(pwd)":/home -v "$(pwd)"/code:/code -v "$(pwd)"/data:/data fsl_test  >> docker_log.txt 2>&1`, it should work with a directory in any location. This would also mean we could use the same setup instructions for pulling the image from dockerhub, where we won't have the github repo locally.

Reply:
ooo that's a much better idea! Shall we leave it as is for now then and update it once the image is on dockerhub?

Reply:
Sounds good to me!

Reply:
Also (side note) - I'm not sure we need all three volumes? /home is probably sufficient by itself, as long as the code and data dirs are always nested inside?